watertap/unit_models/electrodialysis_1D.py

```diff
@@ -775,6 +797,29 @@ def eq_get_velocity_concentrate(self, t, x):
                 == self.concentrate.properties[t, x].flow_vol_phase["Liq"]
             )
 
+        @self.Constraint(
+            self.flowsheet().time,
+            self.membrane_set,
+            self.diluate.length_domain,
+            doc="Calculate membrane areal resistance at L=x",
+        )
+        def eq_get_membrane_areal_resisance_x(self, t, memb, x):
+            return (
+                self.membrane_areal_resistance_x[memb, x]
+                * sum(
+                    self.diluate.properties[t, x].conc_mol_phase_comp["Liq", j]
+                    * self.config.property_package.charge_comp[j]
+                    for j in self.cation_set
```

Comment:
The whole system is electrically neutral but does the diluate stay electrically neutral too?

Reply:
yes